drivers/hyperv/hyperv.go

```diff
@@ -205,21 +210,32 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	log.Infof("Creating VM...")
-	virtualSwitch, err := d.chooseVirtualSwitch()
-	if err != nil {
-		return err
-	}
-
-	log.Infof("Using switch %q", virtualSwitch)
-
-	if err := cmd("Hyper-V\\New-VM",
+	args := []string{
+		"Hyper-V\\New-VM",
 		d.MachineName,
 		"-Path", fmt.Sprintf("'%s'", d.ResolveStorePath(".")),
-		"-SwitchName", quote(virtualSwitch),
-		"-MemoryStartupBytes", toMb(d.Memory)); err != nil {
+		"-MemoryStartupBytes", toMb(d.Memory),
+	}
+	if d.VirtualSwitch != "" {
+		log.Infof("Creating VM...")
```

Comment:
this creating VM message is not part of the switch check.... ?

Reply:
done